bin/elasticsearch.bat

```diff
@@ -1,10 +1,46 @@
 @echo off
 
-SETLOCAL
+SETLOCAL enabledelayedexpansion
 TITLE Elasticsearch ${project.version}
 
+SET params='%*'
+
+:loop
+FOR /F "usebackq tokens=1* delims= " %%A IN (!params!) DO (
+    SET current=%%A
+    SET params='%%B'
+	SET silent=N
+	
+	IF "!current!" == "-s" (
+		SET silent=Y
+	)
+	IF "!current!" == "--silent" (
+		SET silent=Y
+	)	
+	
+	IF "!silent!" == "Y" (
+		SET nopauseonerror=Y
+	) ELSE (
```

Comment:
The option --silent / -s here is intended to avoid pausing when JAVA_HOME or else is not found. What you are suggesting will move the JVM options (think -Des.node.name, -Des.pidfile etc) as program arguments, which is not the same thing.